### DISCUSSION TITLE
Possible to support links without protocol or www?

### DISCUSSION THREAD

**Discussion by iliatapazukk** [remarkjs/remark#916] [Q&A] [answered]
Hi everyone!
Is it possible to autolink url like a `hostname.com` without `http*` and `www.`? 
In a perfect world I would like to get `[hostname.com](hostname.com)` оr `[hostname.com](https://hostname.com)`

**Comment by wooorm** [comment, marked as answer]
Not according to a markdown format we support. But much can be achieved with plugins. Either in the mdast (markdown) or hast (HTML) space.

`remark-github`, which is responsible for linking certain references to GH users/issues/prs/commits, does similar things:

https://github.com/remarkjs/remark-github/blob/f3eb4077db0ab7dbd0cf4473d44fc348f5a3c3af/lib/index.js#L155-L165
